src/lib.rs

```diff
@@ -192,15 +199,27 @@ fn camera_movement(
         Some(c) => Vec2::new(c.x, -c.y),
         None => return,
     };
-    let delta_device_pixels = current_pos - last_pos.unwrap_or(current_pos);
 
-    for (cam, mut transform, projection) in &mut query {
-        if cam.enabled
-            && cam
+    for (entity, cam, pancam_moving, mut transform, projection) in &mut query {
+        if !cam.enabled
+            || !cam
                 .grab_buttons
                 .iter()
                 .any(|btn| mouse_buttons.pressed(*btn))
```

Comment:
Maybe we could simply do pressed(btn) && !just_pressed(btn) ?

Reply:
Good idea ; that said, if we ever want to support multi touch on multiple pancams, we'll want to have pancam independent cursors.

But as it's not the goal of that PR I can use your solution if you prefer.